interfaces/matlab_experimental/Base/ThermoPhase.m

```diff
@@ -1464,6 +1499,9 @@ function display(tp)
         function set.SV(tp, input)
             s = input{1};
             v = input{2};
+            if strcmp(tp.basis, 'molar')
+                s = s/tp.meanMolecularWeight;
+            end
```

Comment:
`basis` also affects the dimensions of the "volume" term, doesn't it? For this and all other setters involving `V`? Likewise, the meaning of `D` changes from mass to molar density, if you want to be consistent with the Python interface.